src/services/FirebaseService.ts

```diff
@@ -86,8 +115,8 @@ export class FirebaseService implements IFirebaseService {
             .storage()
             .bucket(functions.config().extfirebase.bucket)
             .file(`${collectionKey}/${contractAddress}_${fileInfo.name}`);
-        const buffer = Buffer.from(fileInfo.base64content, 'base64');
-        await file.save(buffer, { contentType: fileInfo.contentType });
+        const buffer = Buffer.from(this.decode(fileInfo.base64content), 'base64');
+        await file.save(buffer, { contentType: this.decode(fileInfo.contentType) });
```

Comment:
oh sorry, I didn't point out in the code. I thought here should be catch the exception?

Reply:
No worries, fixed in [this]([5f7a868](https://github.com/AstarNetwork/astar-token-api/pull/48/commits/5f7a868a59a67dbf548423aeddc7bf998fb3c946)) commit

This is double check since all inputs are checked in DappsStakingController